src/test/regress/expected/multi_deparse_procedure.out

```diff
@@ -0,0 +1,316 @@
+--
```

Comment:
same comments as the function test file

missing:
 - DROP PROCEDURE ...
 - DROP PROCEDURE ... CASCADE
 - ALTER PROCEDURE ... SET ... FROM CURRENT
 - ALTER PROCEDURE ... multiple actions